<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicDownloadWidget</class>
 <widget class="QWidget" name="MusicDownloadWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>下载窗口</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>460</width>
     <height>170</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>下载窗口</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>458</x>
     <y>22</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>460</width>
     <height>145</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{color:#666666;background:rgba(255,255,255,200);}
QLabel{background:transparent;}</string>
   </property>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>存储路径：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="downloadPathEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pathChangedButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>更改目录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="settingButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>下载设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downloadButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>下载</string>
    </property>
   </widget>
   <widget class="QLabel" name="downloadName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>2</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:22px;font-weight:bold;</string>
    </property>
    <property name="text">
     <string>XXXXXXXXXXXXXXXXXXX</string>
    </property>
   </widget>
   <widget class="MusicDownloadTableWidget" name="viewArea">
    <property name="geometry">
     <rect>
      <x>101</x>
      <y>33</y>
      <width>330</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>37</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>资源选择：</string>
    </property>
   </widget>
   <zorder>label2</zorder>
   <zorder>downloadName</zorder>
   <zorder>viewArea</zorder>
   <zorder>label1</zorder>
   <zorder>pathChangedButton</zorder>
   <zorder>settingButton</zorder>
   <zorder>downloadPathEdit</zorder>
   <zorder>downloadButton</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicDownloadTableWidget</class>
   <extends>QTableWidget</extends>
   <header>musicdownloadwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
